supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -21,11 +21,11 @@
 
 *See also*:
 
-// BxMS: Added "In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite"
+// BxMS: Added "In Red Hat Process Automation Manager"
 [discrete]
 [[backward-chaining]]
 ==== image:images/yes.png[yes] backward chaining (verb)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, _backward chaining_ is a feature of the rule engine. The backward chaining process is often referred to as derivation queries. It is not as common compared to reactive systems because Red Hat JBoss BRMS is primarily reactive forward chaining, that is, it responds to changes in your data. The backward chaining added to the rule engine is for product-like derivations.
+*Description*: In Red Hat Process Automation Manager, _backward chaining_ is a feature of the rule engine. The backward chaining process is often referred to as derivation queries. It is not as common compared to reactive systems because Red Hat Process Automation Manager is primarily reactive forward chaining, that is, it responds to changes in your data. The backward chaining added to the rule engine is for product-like derivations.
```

Comment:
```suggestion
*Description*: In Red Hat Process Automation Manager, _backward chaining_ is a feature of the rule engine. The backward chaining process is often referred to as _derivation queries_. It is not as common compared to reactive systems because Red Hat Process Automation Manager is primarily reactive forward chaining, that is, it responds to changes in your data. The backward chaining added to the rule engine is for product-like derivations.
```
Since `derivation queries` is presented as an alernative phrase, I think it should italicized. What do you think?